docs/releases/v2022.1.1.rst

```diff
@@ -0,0 +1,64 @@
+Gluon 2022.1
+============
+
+Important notes
+---------------
+
+This release mitigates multiple flaws in the Linux wireless stack causing RCE as well as DoS vulnerabilities.
+
+
+Added hardware support
+----------------------
+
+ath79-nand
+~~~~~~~~~~~~~~
+
+-  ZTE
+
+   -  MF281
+
+rockchip-armv8
+~~~~~~~~~~~~~~
+
+-  FriendlyElec
+
+   -  NanoPi R2S
+
+Bugfixes
+--------
+
+ * Fixes the update path for GL-AR300M devices.
```

Comment:
for `ubiquiti-loco-m-xw` as well
https://github.com/freifunk-gluon/gluon/commit/9667075b540b80d94e35b604728bdb3badfbf768

Reply:
resolved in b33217ebbd31b2413b54ff0d87660c91c0b0a577